ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet05
+    {
+        public int GetNextNumberDivisibleByN(int startNumber, int n)
+        {
+            for (int i = startNumber; i < 10000000; i++)
```

Comment:
Interesting... there'a better option than just an arbitrarily large number. If `i` begins at `startNumber` and you're looking for a multiple of `n`, how much higher than `startNumber` can you go before you would have definitely found one?

On a different note, could you solve this without a loop?